spec/std/file_utils_spec.cr

```diff
@@ -183,6 +183,44 @@ describe "FileUtils" do
     end
   end
 
+  describe "chmod_r" do
+    it "changes the permissions of the directory recursively" do
+      data_path = File.join(__DIR__, "data")
+      path = File.join(data_path, "chmod_r_test")
+
+      begin
+        Dir.mkdir(path)
+        Dir.mkdir(File.join(path, "a"))
+        File.write(File.join(path, "a/b"), "")
+
+        FileUtils.chmod_r(path, 0o775)
+        File.stat(File.join(path, "a")).perm.should eq(0o775)
+        File.stat(File.join(path, "a/b")).perm.should eq(0o775)
+      ensure
+        FileUtils.rm_r(path)
+      end
+    end
+  end
+
+  describe "chown_r" do
+    it "change the owner of the directory recursively" do
+      data_path = File.join(__DIR__, "data")
```

Comment:
If you don't plan on actually running the method, there is no point in setting up the directories.